gateway/config/config.go

```diff
@@ -32,8 +34,11 @@ type Project struct {
 
 // Secret describes the a secret object
 type Secret struct {
-	IsPrimary bool   `json:"isPrimary" yaml:"isPrimary"`
-	Secret    string `json:"secret" yaml:"secret"`
+	IsPrimary  bool          `json:"isPrimary" yaml:"isPrimary"`
+	Secret     string        `json:"secret" yaml:"secret"`
+	Alg        utils.AuthAlg `json:"alg" yaml:"alg"`
```

Comment:
No need to put the type in utils. Put it below this struct. Rename it to `JWTAlg`